src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {movies.map(movie => (
+        <div className="card" data-cy="Movie" key={movie.imdbId}>
+          <MovieCard movie={movie} />
+        </div>
```

Comment:
```suggestion
         <MovieCard movie={movie} key={movie.imdbId} />
```
I suppose, it will be better.